versioning/src/main/scala/org/typelevel/sbt/TypelevelVersioningPlugin.scala

```diff
@@ -82,17 +82,26 @@ object TypelevelVersioningPlugin extends AutoPlugin {
               None
           }
 
+        // version here is the prefix used further to build a final version number
         var version = latestInSeries.fold(tlBaseVersion.value)(_.toString)
 
+        V(version) match {
+          case None =>
+            sys.error(s"version must be semver format: $version")
+          case Some(value) =>
+            if (!(value.isSameSeries(baseV) || value >= baseV))
+              sys.error(
+                s"Your current version $version cannot be less than tlBaseVersion $baseV")
+        }
```

Comment:
I totally missed this: actually, we need to move this check outside of the current `{ }`. Otherwise it's missing the current tags which are read here:
https://github.com/typelevel/sbt-typelevel/blob/b3b9fc2e6aac476260836bc91462468fb9c1a5aa/versioning/src/main/scala/org/typelevel/sbt/TypelevelVersioningPlugin.scala#L66